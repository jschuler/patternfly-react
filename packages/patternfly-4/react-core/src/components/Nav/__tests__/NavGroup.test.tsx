// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { NavGroup } from '../NavGroup';

describe('NavGroup test', () => {
  it('NavGroup should match snapshot', () => {
    const view = shallow(
      <NavGroup
        title={"'defaultString'"}
				children={<>ReactNode</>}
				className={"''"}
				id={"string"}
      />);
    expect(view).toMatchSnapshot();
  });
});
