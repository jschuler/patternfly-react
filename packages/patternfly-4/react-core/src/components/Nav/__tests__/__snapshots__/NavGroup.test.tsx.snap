// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavGroup test NavGroup should match snapshot 1`] = `
<section
  aria-labelledby="'defaultString'"
  className="pf-c-nav__section ''"
>
  <h2
    className="pf-c-nav__section-title"
    id="'defaultString'"
  >
    'defaultString'
  </h2>
  <ul
    className="pf-c-nav__list"
  >
    ReactNode
  </ul>
</section>
`;
